README.md

```diff
@@ -10,6 +17,92 @@ To build this snap locally you need snapcraft. The project must be built as real
 $ sudo snapcraft
 ```
 
+# Testing with spread
+
+## Prerequisites for testing
+
+You need to have the following software installed before you can test with spread
+ - Go (https://golang.org/doc/install or ```sudo snap install go```)
+ - Spread (install from source as per below)
+
+## Installing spread
+
+Manually installing spread is the recommended way to get started as
+we need some LXD profile support that has not reached upstream. This document will be updated with the upstream version when this happens.
+
+```
+git clone https://github.com/Meulengracht/spread
```

Comment:
Maybe you could mention that you can install with `snap install spread`, but that if you want to run with the lxd backend you would need to follow these steps.

Reply:
This is a really good point - I'll update the documentation.